google/cloud/spanner/samples/samples.cc

```diff
@@ -4610,6 +4877,13 @@ int RunOneCommand(std::vector<std::string> argv) {
       {"list-backup-operations", ListBackupOperationsCommand},
       {"list-database-operations", ListDatabaseOperationsCommand},
       {"update-database", UpdateDatabaseCommand},
+      {"create-backup-schedule", CreateBackupScheduleCommand},
+      {"create-incremental-backup-schedule", CreateIncrementalBackupScheduleCommand},
+      {"create-backup-schedule-with-encryption", CreateBackupScheduleWithEncryptionCommand},
```

Comment:
our formatter thinks these lines are too long. To defeat the CI, run:

```
ci/cloudbuild/build.sh -t checkers-pr
```

Reply:
Done.